src/lib.rs

```diff
@@ -237,7 +237,7 @@
 #[cfg(feature="std")] extern crate std as core;
 #[cfg(all(feature = "alloc", not(feature="std")))] extern crate alloc;
 
-#[cfg(test)] #[cfg(feature="serde1")] extern crate bincode;
+#[cfg(all(feature="serde1", test))] extern crate bincode;
```

Comment:
This comment does not really belong to this PR, but does rand still need bincode as dependency, or do all tests now live in external crates?